src/pages/academy/components/video-banner/carousel/_carousel-style.ts

```diff
@@ -66,18 +66,18 @@ export const StyledButtonWrapper = styled.div`
     }
 `
 
-export const StyledChevron = styled((props) => <Chevron {...props} />)`
+export const StyledChevron = styled(Chevron)<{ color?: string }>`
```

Comment:
Can you check if we can use `React.CSSProperties` here instead

Reply:
Did it 👍